src/control/system/rank.go

```diff
@@ -147,3 +147,21 @@ func RanksFromUint32(ranks []uint32) (sysRanks []Rank) {
 
 	return
 }
+
+// TestRankMembership compares two Rank slices and returns a
+// Rank slice with any ranks found in the second slice that do
+// not exist in the first slice.
+func TestRankMembership(members, toTest []Rank) (missing []Rank) {
```

Comment:
naming nit - I might call this "CheckRankMembership" instead because of the naming conventions we use for unit test-related functions

Reply:
Done.